CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# v0.6.0 (Unreleased)
+### FEATURES
```

Comment:
I might just make a Deprecation header or something for this, since it's not really a feature.